ide/db.sql.editor/src/org/netbeans/modules/db/sql/editor/completion/SQLCompletionProvider.java

```diff
@@ -131,4 +264,93 @@ public void run() {
         TokenHierarchy<?> hierarchy = TokenHierarchy.get(doc);
         return hierarchy.tokenSequence(SQLTokenId.language());
     }
+        
+    private static class SqlNewConnectionFix implements Fix {
```

Comment:
This feels a bit strange. We don't put "Install JDK" options into the Java Sourcecode. Why is SQL different?

Reply:
At the time I was trying to make a "fix" to do the same as setting the connection in the upper connection bar.  Some of these fixes are really just open other dialogs (like drivers or new connection) as present already so maybe these hint/fixes are not really relevant.

Reply:
I've removed all "hint/fixes" in recent commit